front/app/containers/Admin/reporting/components/ReportBuilder/TopBar/ReportTitle.tsx

```diff
@@ -73,23 +75,33 @@ const ReportTitle = ({ reportId }) => {
       if (reportTitleIsTaken(error)) {
         showError(formatMessage(messages.titleTaken));
       }
+
+      throw error;
```

Comment:
Why do you reraise the error, then catch and ignore it?

Reply:
If we don't throw an error here, and if we wouldn't use `try/catch` in `handleOnKeyDown`, the `inputRef?.blur()` would always be called after `await updateTitle()` in `handleOnKeyDown`. By throwing an error here and using `try` in `handleOnKeyDown`, we interrupt code executing after `await updateTitle()` in case of an error so that `inputRef?.blur()` never gets called.